stock_prices/stock_prices.py

```diff
@@ -3,7 +3,16 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  profits = []
+  for i in enumerate(prices):
+    index = i[0]
+    if index < len(prices)-2:
+      for price in prices:
+        if prices.index(price) >= index+1:
+          profits.append(price - prices[index])
+
+  profits.sort(reverse = True)
```

Comment:
Remember, `sort()` is an expensive operation. Can you think of a way where we can avoid it?
(_Hint: We don't care about any profit value except the highest one. Does your program need to remember profit values other than the one we care about?_).